policies/paying-for-things-and-getting-reimbursed.md

```diff
@@ -34,6 +34,7 @@ All wire transfers require the following information:
 * address of their bank
 * SWIFT code of their bank (if applicable)
 * currency of their account
+* paypal payment, voucher
```

Comment:
```suggestion

```

Reply:
That would be for paypal requests only. There should be an issue that adds instructions for PayPal and WISE (the other two repayment methods we support)

Reply:
Okay, I will look into it also how do I make an issue?  I have never done it before

Reply:
https://github.com/BlackPythonDevs/blackpythondevs/issues/new

Add information like:
- What is the current issue
- (If Possible) how can this issue be resolved

Reply:
Sorry for my late respond I was not feeling well and had exams. I have created the issue and the changes needed, I saw that they were made already , should I still do them or leave them